package.json

```diff
@@ -6,7 +6,7 @@
   "homepage": "https://github.com/SocialGouv/code-du-travail-numerique#readme",
   "scripts": {
     "build": "lerna run build",
-    "build:ui": "yarn workspace @socialgouv/react-fiche-service-public build && yarn workspace @socialgouv/react-ui build",
```

Comment:
wait... its not in alphabetical order :trollface:

Reply:
Ah oui mais là si tu build les fiches sp avant l'ui alors que t'as modifié le deux tu vas avoir des surprises !